src/Eccube/Form/Type/Admin/MasterdataType.php

```diff
@@ -60,6 +62,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 foreach ($classNames as $className) {
                     /** @var ClassMetadata $meta */
                     $meta = $this->entityManager->getMetadataFactory()->getMetadataFor($className);
+
+                    // OrderStatus/OrderStatusColorは対象外
+                    // @see https://github.com/EC-CUBE/ec-cube/pull/4844
+                    if (in_array($meta->getName(), [OrderStatus::class, OrderStatusColor::class])) {
```

Comment:
```suggestion
                    if (in_array($meta->getName(), [OrderStatus::class, CustomerOrderStatus::class, OrderStatusColor::class])) {
```

`use Eccube\Entity\Master\CustomerOrderStatus;`も追加する必要あり

Reply:
追加しました